### PR TITLE
Udated backends and randomized user-agent

### PR DESCRIPTION
**Pull request by induane**
Avoid getting hit with a captcha

### CODE REVIEW

File: src/dopameme/search.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def __init__(self, **kwargs):
         self.modifier = kwargs.get("modifier", rand_mod) or rand_mod
 
     @property
-    def default_str(self):
+    def default_str(self) -> str:
```
Comment (middlemarch) [inline src/dopameme/search.py:35]: This is why I'm on the fence about Typing, the clue is in the name, this definitely returns a string! If your function doesn't tell you what it does, ask it again.

File: src/dopameme/constants.py [hunk 2]
```diff
@@ -113,12 +112,15 @@
...
     "etsy",
     "goldporntube",
+    "xvideos",
     "google.com/search",
     "item",
     "pornhub",
     "pornography",
     "product",
     "redtube",
+    "spermyporn",  # Hilarious but why does this keep coming up on bing?
```
Comment (middlemarch) [inline src/dopameme/constants.py:122]: Gates likes spermyporn?!

**Review by middlemarch — APPROVED** [review]
This took me waaaay too long, apologies! :flying_saucer:

File: src/dopameme/search.py (hunk 1 above)
Comment (induane) [reply src/dopameme/search.py:35]: The point of it for the most part is to enable static analysis of the code in order to catch bugs without having to run a program.

I go both ways with it but I'm mostly in favor after a while of using them.

**Pull request merged**